emhttp/plugins/dynamix/include/SysDevs.php

```diff
@@ -170,9 +204,9 @@ function usb_physical_port($usbbusdev) {
           echo (in_array($pciaddress."|".$vd, $vfio_cfg_devices) || in_array($pciaddress, $vfio_cfg_devices)) ? " checked>" : ">";
           } 
         } else { echo "</td><td>"; }
-        echo '</td><td title="';
+        echo '</td><td style="padding-right:12px;" title="';
```

Comment:
For the new responsive GUI, styling should not be in the main code but should be in the styling page in the sheet directory.